bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -648,6 +656,15 @@ type BottlerocketBootstrapContainer struct {
 	UserData string `json:"userData,omitempty"`
 }
 
+// CertBundle holds the cert data.
+type CertBundle struct {
+	// Name is the name of the cert bundle.
+	Name string `json:"name"`
```

Comment:
Are users still expected to pass names for the certs? Wasn't there an issue with that if they used the same name for all their certs?
I thought we were going to use randomly generated names for certs since they don't really matter?

Reply:
Yeah, users are still going to be expected to pass the names for the certs. I'll add a preflight check to make sure users cannot pass the same name multiple times.